src/data.rs

```diff
@@ -101,7 +101,7 @@ pub struct Data {
 
 impl Data {
     pub fn new(
-        date: OffsetDateTime,
+        date: Option<OffsetDateTime>,
```

Comment:
Maybe add another constructor instead, so that we don't break API?

This would allow having an explicit naming like "new_with_ingestion_timestamp" or something like that

Reply:
Of course!

Reply:
I added a new function to create a data object without time :D